monkey/monkey_island/cc/next_ui/server.js

```diff
@@ -5,14 +5,17 @@ const https = require('https')
 const {parse} = require('url');
 
 const fs = require('fs');
-const port = 443
-const app = next({dev: false, port: port});
+
+const config = fs.readFileSync('../server_config.json')
```

Comment:
I think this will break as `server_config.json` file is in different directory on docker

Reply:
It is? How does this work then?
https://github.com/guardicore/monkey/blob/68450b9e31b5aca856f370813585062a1c068487/monkey/monkey_island/cc/setup/config_setup.py#L12-L24

Reply:
It reads island config path from cmd arguments. Currently nothing passes this arg to the next.js server, so it won't know where to find the config